Rakefile

```diff
@@ -20,6 +21,9 @@ Hoe.spec "flay" do
   dependency "ruby_parser",    "~> 3.0"
   dependency "erubis",         "~> 2.7.0"
 
+  dependency "minitest",       "~> 5.8.0", :dev
+  dependency "ruby2ruby",      "~> 2.2.0", :dev
```

Comment:
ruby2ruby is _technically_ optional and rubygems doesn't have optional deps.

minitest should come from having the hoe-seattlerb plugin gem installed.

Some of us (\*cough* ok me) don't use bundler.